src/elements/button/button.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 angular
-  .module('semantic.ui.elements.button', [])
+  .module('semantic.ui.elements.button', ['semantic.ui.elements.icon', 'semantic.ui.elements.divider'])
```

Comment:
Changes that you made in _**'semantic.ui.elements.button'**_ module are not related to your initial PR and _should not_ be here